ukvm/ukvm_guest.h

```diff
@@ -264,4 +264,10 @@ struct ukvm_poll {
     int ret;
 };
 
+/* UKVM_HYPERCALL_HALT */
+struct ukvm_halt {
+    /* OUT */
```

Comment:
Shouldn't this be `IN` ?

Reply:
Yes, it should be an IN, just changed it.